### DISCUSSION TITLE
How should BoxedWine decide which exe to run from a zip?

### DISCUSSION THREAD

**Discussion by hello-smile6** [DustinBrett/daedalOS#183] [Feature Requests]
I want to try running Chromium, but BoxedWine chooses the wrong .exe file.

**Comment by DustinBrett** [comment]
Good point, that is indeed not ideal. I'll try and think of a simple solution. Possibly a custom shortcut could make this easier.

**Comment by DustinBrett** [comment]
If I had a file selection dialog it would be easier, but I don't and I don't desire to use something basic like `dialog()`. I'd rather there be some logic. I've moved this out of being a bug as it's known and there just isn't a better coded behavior for it. When I have time I will eventually address this. BoxedWine could use a few upgrades.
